langkit/config.py

```diff
@@ -0,0 +1,10 @@
+import os
+
+# Cache location
+DEFAULT_XDG_CACHE_HOME: str = "~/.cache"
+XDG_CACHE_HOME: str = os.getenv("XDG_CACHE_HOME", DEFAULT_XDG_CACHE_HOME)
+DEFAULT_CACHE_HOME: str = os.path.join(XDG_CACHE_HOME, "langkit")
+
+LANGKIT_CACHE: str = os.path.expanduser(os.getenv("LANGKIT_CACHE", DEFAULT_CACHE_HOME))
+
+LANGKIT_INJECTIONS_CACHE: str = os.path.join(LANGKIT_CACHE, "injections")
```

Comment:
nice, this looks great. I think this one line can be moved into the injections module, but the rest can stay here for the moment.